parsebib.el

```diff
@@ -240,113 +240,144 @@ trailing space will be included in the overall match."
-         ("d"  "\N{COMBINING DOT BELOW}")
-         ("H"  "\N{COMBINING DOUBLE ACUTE ACCENT}")
-         ("k"  "\N{COMBINING OGONEK}")
-         ("U"  "\N{COMBINING DOUBLE VERTICAL LINE ABOVE}")
-         ("u"  "\N{COMBINING BREVE}")
-         ("v"  "\N{COMBINING CARON}")
-         ("~"  "\N{COMBINING TILDE}")
-         ("|"  "\N{COMBINING COMMA ABOVE}")
-         ("f"  "\N{COMBINING INVERTED BREVE}")
-         ("G"  "\N{COMBINING DOUBLE GRAVE ACCENT}")
-         ("h"  "\N{COMBINING HOOK ABOVE}")
-         ("C"  "\N{COMBINING DOUBLE GRAVE ACCENT}")
-         ("r"  "\N{COMBINING RING ABOVE}")))
-
-    ;; LaTeX2 Escapable "Special" Characters
-    ("\\\\%" . "%") ("\\\\&" . "&") ("\\\\#" . "#") ("\\\\\\$" . "$")
+  (upcase str))
+
+(defvar parsebib-TeX-command-replacement-alist
+  '(("ddag" . "\N{DOUBLE DAGGER}")
+    ("textdaggerdbl" . "\N{DOUBLE DAGGER}")
+    ("dag" . "\N{DAGGER}")
+    ("textdagger" . "\N{DAGGER}")
+    ("textpertenthousand" . "\N{PER TEN THOUSAND SIGN}")
+    ("textperthousand" . "\N{PER MILLE SIGN}")
+    ("textquestiondown" . "\N{INVERTED QUESTION MARK}")
+    ("P" . "\N{PILCROW SIGN}")
+    ("textdollar" . "$")
+    ("S" . "\N{SECTION SIGN}")
+    ("ldots" . "\N{HORIZONTAL ELLIPSIS}")
+    ("dots" . "\N{HORIZONTAL ELLIPSIS}")
+    ("textellipsis" . "\N{HORIZONTAL ELLIPSIS}")
+    ("textemdash" . "\N{EM DASH}")
+    ("textendash" . "\N{EN DASH}")
 
+    ;; Non-ASCII Letters (Excluding Accented Letters)
+    ("AA" . "\N{LATIN CAPITAL LETTER A WITH RING ABOVE}")
+    ("AE" . "\N{LATIN CAPITAL LETTER AE}")
+    ("DH" . "\N{LATIN CAPITAL LETTER ETH}")
+    ("DJ" . "\N{LATIN CAPITAL LETTER ETH}")
+    ("L"  . "\N{LATIN CAPITAL LETTER L WITH STROKE}")
+    ("SS" . "\N{LATIN CAPITAL LETTER SHARP S}")
+    ("NG" . "\N{LATIN CAPITAL LETTER ENG}")
+    ("OE" . "\N{LATIN CAPITAL LIGATURE OE}")
+    ("O"  . "\N{LATIN CAPITAL LETTER O WITH STROKE}")
+    ("TH" . "\N{LATIN CAPITAL LETTER THORN}")
+
+    ("aa" . "\N{LATIN SMALL LETTER A WITH RING ABOVE}")
+    ("ae" . "\N{LATIN SMALL LETTER AE}")
+    ("dh" . "\N{LATIN SMALL LETTER ETH}")
+    ("dj" . "\N{LATIN SMALL LETTER ETH}")
+    ("l"  . "\N{LATIN SMALL LETTER L WITH STROKE}")
+    ("ss" . "\N{LATIN SMALL LETTER SHARP S}")
+    ("ng" . "\N{LATIN SMALL LETTER ENG}")
+    ("oe" . "\N{LATIN SMALL LIGATURE OE}")
+    ("o"  . "\N{LATIN SMALL LETTER O WITH STROKE}")
+    ("th" . "\N{LATIN SMALL LETTER THORN}")
+
+    ("ij" . "ij")
+    ("i" . "\N{LATIN SMALL LETTER DOTLESS I}")
+    ("j" . "\N{LATIN SMALL LETTER DOTLESS J}")
+    ;; Formatting Commands
+    ("textit" . parsebib--convert-tex-italics)
+    ("emph"   . parsebib--convert-tex-italics)
+    ("textbf" . parsebib--convert-tex-bold)
+    ("textsc" . parsebib--convert-tex-small-caps))
+  "An alist of COMMAND-REPLACEMENT pair for a LaTeX command.
+
+COMMAND is the name of a TeX or LaTeX command (without
+backslash).  REPLACEMENT can be either a string or a
+function. If it is a function, it is passed the first
```

Comment:
There should be two spaces after a full stop. (Use checkdoc to look for issues with doc strings.)

Reply:
Checkdoc does turn up anything for me but I fixed the cases I found. Hopefully all.